src/coreclr/jit/codegenarm64test.cpp

```diff
@@ -4802,11 +4802,11 @@ void CodeGen::genArm64EmitterUnitTestsSve()
                               INS_OPTS_SCALABLE_D); // CLASTB  <Zdn>.<T>, <Pg>, <Zdn>.<T>, <Zm>.<T>
 
     // IF_SVE_CN_3A
-    theEmitter->emitIns_R_R_R(INS_sve_clasta, EA_2BYTE, REG_V12, REG_P1, REG_V15, INS_OPTS_SCALABLE_H,
```

Comment:
Same changes as for `AddAcross` in previous PR - the `size` arg is not used, as the sizes are dependant on `opts`.